src/interactive/Modal.jsx

```diff
@@ -99,6 +100,10 @@ class Modal extends React.Component {
 			className,
 			children,
 			fullscreen,
+			heroBgColor,
+			heroBgImage,
+			heroContent,
+			heroInverted,
```

Comment:
I like all the other prop names, but for inverted context, I'd stick with `inverted` instead of `heroInverted`. The `inverted` prop is present on a number of components and it would be good to keep it consistent.